buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -486,6 +486,7 @@ class BeamModulePlugin implements Plugin<Project> {
     def jsr305_version = "3.0.2"
     def everit_json_version = "1.14.1"
     def kafka_version = "2.4.1"
+    def log4j2_version = "2.19.0"
```

Comment:
Is there additional risk added by bumping the version? It was 2.17.1 before, not sure if this small version bump will be a problem. I think 2.17.1ish is in the Dataflow legacy worker container, will this conflict with that?

Reply:
Dataflow legacy uses 2.17.2 which I swapped to here. Upgrading can happen in a separate PR.